src/input.ts

```diff
@@ -32,7 +41,16 @@ export class Input {
     this.repo = validatedParams.repo
     this.packageName = validatedParams.packageName
     this.numOldVersionsToDelete = validatedParams.numOldVersionsToDelete
+    this.minVersionsToKeep = validatedParams.minVersionsToKeep
+    this.ignoreVersions = validatedParams.ignoreVersions
+    this.deletePreReleaseVersions = validatedParams.deletePreReleaseVersions
     this.token = validatedParams.token
+
+    if (this.deletePreReleaseVersions == 'true') {
+      this.numOldVersionsToDelete = 100
```

Comment:
why we are setting it to 100? what if user wants to delete more versions. we are ignoring this input when deletePreReleaseVersions is set.

Reply:
The graphql api has a limit to get only 100 records. On increasing this limit the api call results in an error.

Reply:
ok got it